tests/foreman/ui/test_settings.py

```diff
@@ -180,12 +180,12 @@ def test_positive_host_dmi_uuid_duplicates(session, setting_update):
 
     :CaseImportance: High
     """
-    property_value = f'[ {gen_string("alpha")} ]'
+    property_value = gen_string("alpha")
     property_name = setting_update.name
     with session:
         session.settings.update(f'name = {setting_update.name}', property_value)
         result = session.settings.read(f'name = {property_name}')
-        assert result['table'][0]['Value'] == property_value
+        assert result['table'][0]['Value'] == f'[{property_value}]'
```

Comment:
I think no need to pass the settings in the square bracket check the last comment on the 1975713.

Reply:
@devendra104 There is an issue with the airgun implementation which reads the data from PopOver widget so it reads as a text. To make this change for just one test case might cause issue so I modified in test case itself.